echo-pipelinetriggers/src/main/java/com/netflix/spinnaker/echo/pipelinetriggers/eventhandlers/GitEventHandler.java

```diff
@@ -79,13 +76,12 @@ protected boolean isValidTrigger(final Trigger trigger) {
   }
 
   @Override
-  protected boolean isSuccessfulTriggerEvent(TriggerEvent event) {
+  public boolean isSuccessfulTriggerEvent(GitEvent event) {
     return true;
   }
 
   @Override
-  protected Predicate<Trigger> matchTriggerFor(final TriggerEvent event, final Pipeline pipeline) {
-    GitEvent gitEvent = (GitEvent) event;
+  protected Predicate<Trigger> matchTriggerFor(GitEvent gitEvent, final Pipeline pipeline) {
```

Comment:
Why no longer `final`?

Reply:
No specific reason, other than it seemed reasonably low-value to enforce that the function can't re-asssign another pipeline to the local variable `pipeline`.  The code in this module was marking parameters as `final` in a lot (but not all) cases where the parameter was not re-assigned.  Given that it doesn't seem to be the general style in our codebase to mark parameters a final, I leaned towards just removing the cases where things were.  (Though I could see the argument to explicitly put `final` everywhere.)